libs/markets/src/lib/components/market-info/tooltip-mapping.tsx

```diff
@@ -106,4 +106,47 @@ export const tooltipMapping: Record<string, ReactNode> = {
   insurancePoolFraction: t(
     'The fraction of the insurance pool balance that is carried over from the parent market to the successor.'
   ),
+  priceRange: t('The price range of the market. '),
+  commitmentMinTimeFraction: t(
+    `The minimum time fraction for a commitment. Specifies the minimum fraction of time LPs must spend 'on the book' providing their committed liquidity.`
+  ),
+  providersFeeCalculationTimeStep: t(
+    'The time step for providers fee. Specifies how often the quality of liquidity supplied by the LPS is evaluated and fees arising from that period are earmarked for specific parties.'
+  ),
+  performanceHysteresisEpochs: t(
+    'The performance hysteresis epochs. Specifies the number of liquidity epochs over which past performance will continue to affect rewards.'
+  ),
+  SLACompletionFactor: t(
+    `The SLA completion factor. Specifies the maximum fraction of their accrued fees an LP that meets the SLA implied by market.liquidity.commitmentMinTimeFraction will lose to liquidity providers that achieved a higher SLA performance than them. `
+  ),
+  bondPenaltyParameter: t(
+    'used to calculate the penalty to liquidity providers when they fail to support their open position through sufficient general+margin balance. Valid values: any decimal number >= 0 with a default value of 0.1.'
+  ),
+  nonPerformanceBondPenaltySlope: t(
+    'used to calculate how much is the LP bond slashed if they fail to reach the minimum SLA. Valid values: any decimal number >= 0 with a default value of 2.0.'
+  ),
+  nonPerformanceBondPenaltyMax: t(
+    'used to calculate how much is the LP bond slashed if they fail to reach the minimum SLA. Valid values: any decimal number >= 0 and <=1.0 with a default value of 0.5.'
+  ),
+  maximumLiquidityFeeFactorLevel: t(
+    'used in validating fee amounts that are submitted as part of the LP commitment transaction. Note that a value of 0.05 = 5%. Valid values are: any decimal number >=0 and <=1. Default value 1.'
+  ),
+  stakeToCCYVolume: t(
+    'used to translate a commitment to an obligation. Any decimal number >0 with default value 1.0.'
+  ),
+  epochLength: t(
+    'LP rewards from liquidity fees are paid out once per epoch according to whether they met the "SLA" (implied by market.liquidity.commitmentMinTimeFraction) and their previous performance (for the last n epochs defined by market.liquidity.performanceHysteresisEpochs), see epoch spec.'
+  ),
+  earlyExitPenalty: t(
+    '(decimal ≥0), sets how much LP forfeits of their bond in case the market is below target stake and they wish to reduce their commitment. If set to 0 there is no penalty for early exit, if set to 1 their entire bond is forfeited if they exit their entire commitment, if set >1, their entire bond will be forfeited for exiting 1/earlyExitPenalty of their commitment amount.'
```

Comment:
```suggestion
    'Sets how much an LP forfeits of their bond if they reduce their commitment while the market is below target stake. If set to 0 there is no penalty for early exit, if set to 1 their entire bond is forfeited if they exit their entire commitment, if set >1, their entire bond will be forfeited for exiting 1/earlyExitPenalty of their commitment amount. Must be a decimal ≥0. '
```

Reply:
I personally don't find this part "if set >1, their entire bond will be forfeited for exiting 1/earlyExitPenalty of their commitment amount." easy to understand at a glance. Could you simplify?